rkfood_app/views/fb_views.py

```diff
@@ -138,78 +137,96 @@ def send_otp_via_email_or_sms(user:User):
     #     to=user.phone_number
     # )
 
-# def customer_login(request):
-#     if request.method == 'POST':
-#         if 'otp_request' in request.POST:
-#             otp_form = OTPRequestForm(request.POST)
-#             if otp_form.is_valid():
-#                 email_or_phone = otp_form.cleaned_data.get('email_or_phone', None)
-#                 user = User.objects.get(email=email_or_phone)
-#                 send_otp_via_email_or_sms(user)
-#                 request.session['user_id'] = user.id
-#                 return redirect('')
-#
-
+from django.db import transaction
 
 def customer_register(request):
     if request.method == 'POST':
-        username: str = request.POST.get('username', None).strip()
-        password: str = request.POST.get('password', None).strip()
-        email: str = request.POST.get('email', None).strip()
-        phone: str = request.POST.get('phone', None).strip()
+        username = request.POST.get('username', None).strip()
```

Comment:
if username is not lowercase strictly restrict user to make sure that they enter lowercase input